bundles/binding/org.openhab.binding.zwave/database/fibaro/fgs222.xml

```diff
@@ -113,20 +113,23 @@
 			<Type>list</Type>
 			<Default>1</Default>
 			<Size>1</Size>
-			<Label lang="en">Control key #2 behaviour</Label>			
+			<Label lang="en">Sending commands to control devices assigned to 2-nd association group (key no. 2)</Label>			
 			<Item>
 				<Value>0</Value>
-				<Label lang="en">Device status is not checked</Label>
+				<Label lang="en">Commands are sent when device is turned on and </Label>
 			</Item>
 			<Item>
 				<Value>1</Value>
-				<Label lang="en">Device status is checked</Label>
+				<Label lang="en">![CDATA[Commands are sent when device is turned off. Enabling device does not send control commands.<BR />
```

Comment:
Likewise, I think this is too long for a label. I'm also not sure it's a good idea to have HTML in the label like this - it's probably best to keep the longer HTML descriptions in the help?